### ISSUE TITLE
Could you please make a version for 1.21?

### ISSUE THREAD

**Issue by Ladiesman287** [CammiesCorner/Cammies-Minecart-Tweaks#52]
This mod looks so good, and we wanted to try it on our server, but there is no 1.21 version :((((

**Comment by j0schi** [comment]
+1 :)

**Comment by CammiePone** [comment]
yeah, it'll be a little bit but i intend to when i can get around to it. it'll also be for neoforge and fabric when i do that

**Comment by ajaxnotmeta** [comment]
i hope you're okay(and hopefully better helps specifically for 1.22 that might be relased on sumer next year) since i mainly uses this mod during 1.19 era and miss it for 2 major versions during your rest from modding minecraft(and sadly despite i want to help, i cant read java 😢 )

**Comment by CammiePone** [comment]
no worries. im doing better now and i might work on it over the coming weeks. there's a lot that needs overhauling

**Comment by ajaxnotmeta** [comment]
good to know! also quick question, is it possible to turn off certain features in the future? since i plan to put this as a server side only(as theres no serverside minecart tweaks that works quite well with furnace minecart, like make the chain invisible for people who dont install the mod client side)

**Comment by NealSavage** [comment]
Hoping we get this for 1.21.4 on Fabric! Been a few years since I've played minecraft, but I'm sure you remember me from the help I gave you when you were first starting this mod hahaha

**Comment by NealSavage** [comment]
> i hope you're okay(and hopefully better helps specifically for 1.22 that might be relased on sumer next year) since i mainly uses this mod during 1.19 era and miss it for 2 major versions during your rest from modding minecraft(and sadly despite i want to help, i cant read java 😢 )

Hahahaha same! Would help if I could, but sadly can't but still need this mod to be functional again!

**Comment by YuiR70** [comment]
hey! any news on a 1.21 port? this mod is really cool, and I'd love to use it, but I'm currently running 1.21.1 :((

also a cool feature you could add is making minecarts able to travel through unloaded chunks

**Comment by ajaxnotmeta** [comment]
> hey! any news on a 1.21 port? this mod is really cool, and I'd love to use it, but I'm currently running 1.21.1 :((
> 
> also a cool feature you could add is making minecarts able to travel through unloaded chunks

if i recall this feature already present in the older version
